interfaces/handler/project.go

```diff
@@ -12,17 +12,17 @@ import (
 )
 
 type ProjectHandler struct {
-	service service.ProjectService
+	s service.ProjectService
 }
 
 func NewProjectHandler(s service.ProjectService) *ProjectHandler {
-	return &ProjectHandler{service: s}
+	return &ProjectHandler{s: s}
```

Comment:
まあ好みなんですが、traPortfolioではこっちの方が多そう
```suggestion
	return &ProjectHandler{s}
```